recipe/yum_requirements.txt

```diff
@@ -1,9 +1 @@
 mesa-libGL
```

Comment:
`mesa` used to in our dockerimage in the past and apparently we still don't have a clear way to use a `ctd` for it :-(